packages/synapse-react-client/src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -68,25 +73,21 @@ export default function DownloadDetails(props: DownloadDetailsProps) {
   return (
     <span className="download-details-container">
       <span>
-        <span className={fileCountIconClass}>
-          <IconSvg icon="file" />
-        </span>
+        <InsertDriveFileTwoTone className={fileCountIconClass} />
```

Comment:
The wrapping span had extra margin applied to it by \_download-list.scss. Use a MUI icon so that the classNames can be passed directly to the icon and the wrapping span can be removed